frontend/src/Components/LegalNoticeContainer/LegalNoticeContainer.tsx

```diff
@@ -27,7 +27,7 @@ export const LegalNoticeContainer = () => {
       <a href="#" onClick={expand}>
         privacy&nbsp;notice&nbsp;&amp;&nbsp;imprint
       </a>
-      &nbsp;(German)
+      &nbsp;(English)
```

Comment:
Can we skip this info, now that the hole app is in English?

Reply:
no strong opinion

Reply:
removed